File: backend/LookApp.Database/Models/Category.cs
using System.Collections.Generic;

namespace LookApp.Database.Models
{
    public class Category
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string UnitOfMeasure { get; set; }
        public double? LowerLimit { get; set; }
        public double? UpperLimit { get; set; }

        public ICollection<Record> Records { get; set; }

        public int CreatorId { get; set; }
        public User Creator { get; set; }
    }
}

File: frontend/dashboard/src/app/shared/models/category.ts
import {RecordModel} from './record';
import { User } from './user';

export class CategoryModel{
  id: number;
  title: string;
  description: string;
  unitOfMeasure: string;
  lowerLimit?: number;
  upperLimit?: number;
  records: RecordModel[];
  creatorId: number;
  creator?: User;
}
